lua/nvim-tree.lua

```diff
@@ -373,6 +402,7 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
   open_on_setup_file = false,
   open_on_tab = false,
   sort_by = "name",
+  root_dirs = { "/usr/include" },
```

Comment:
i must ask though, why adding /usr/include by default ? I'm on nixos and this directory does not even exist, people on windows will also not have it. I'd prefer keeping this table empty by default.

Reply:
yes empty table is better. Updated.

Reply:
Fair enough. I didn't think of other OSs ;)

Reply:
This is very useful:

```lua
		root_dirs = { "/usr/include/yaml-cpp", "/usr/include/linux", "/usr/include", },
```